src/fides/api/schemas/privacy_preference.py

```diff
@@ -1,334 +1,10 @@
-        description="If supplied, TC strings and AC strings are decoded and preferences saved for purpose_consent, "
-        "purpose_legitimate_interests, vendor_consent, vendor_legitimate_interests, and special_features"
-    )
-    preferences: conlist(ConsentOptionCreate, max_items=10000) = []  # type: ignore
-    special_purpose_preferences: conlist(TCFSpecialPurposeSave, max_items=10000) = []  # type: ignore
-    feature_preferences: conlist(TCFFeatureSave, max_items=10000) = []  # type: ignore
-    system_consent_preferences: conlist(TCFVendorSave, max_items=10000) = []  # type: ignore
-    system_legitimate_interests_preferences: conlist(TCFVendorSave, max_items=10000) = []  # type: ignore
-    policy_key: Optional[FidesKey]  # Will use default consent policy if not supplied
-    privacy_experience_id: Optional[SafeStr]
-    user_geography: Optional[SafeStr]
-    method: Optional[ConsentMethod]
-
-    @root_validator()
-    @classmethod
-    def validate_tcf_attributes(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        """
-        Run a few validation checks related to saving privacy preferences against TCF attributes.
-
-        - Check that there are no duplicates preferences against TCF components
-        to avoid confusing responses
-        - Make sure that if a TC string is supplied, we're not also supplying values for the related
-        TC sections individually, because the TC string will override everything here.
-        """
-
-        def tcf_duplicates_detected(preference_list: List) -> bool:
-            identifiers = [preference.id for preference in preference_list]
-            return len(identifiers) != len(set(identifiers))
-
-        for field_name in TCF_PREFERENCES_FIELD_MAPPING:
-            if tcf_duplicates_detected(values.get(field_name, [])):
-                raise ValueError(
-                    f"Duplicate preferences saved against TCF component: '{field_name}'"
-                )
-
-        if values.get("fides_string"):
-            for field in FidesStringFidesPreferences.__fields__:
-                if values.get(field):
-                    raise ValueError(
-                        f"Cannot supply value for '{field}' and 'fides_string' simultaneously when saving privacy preferences."
-                    )
-
-        return values
-
-
-class PrivacyPreferencesCreate(PrivacyPreferencesRequest):
-    """Schema for creating privacy preferences that is supplemented with information
-    from the request headers and the experience"""
-
-    anonymized_ip_address: Optional[str]
-    experience_config_history_id: Optional[SafeStr]
-    request_origin: Optional[RequestOrigin]
-    url_recorded: Optional[SafeStr]
-    user_agent: Optional[SafeStr]
 
 
 class MinimalPrivacyPreferenceHistorySchema(FidesSchema):
     """Minimal privacy preference history schema for building consent emails"""
 
     preference: UserConsentPreference
     privacy_notice_history: PrivacyNoticeHistorySchema
```

Comment:
Most of the Privacy Preference schemas have moved to Plus for convenience.  This stays here because this is for building consent emails.